assembly/src/assembler/module_graph/mod.rs

```diff
@@ -326,14 +476,32 @@ impl ModuleGraph {
 
     /// Fetch a [Module] by [ModuleIndex]
     #[allow(unused)]
-    pub fn get_module(&self, id: ModuleIndex) -> Option<Arc<Module>> {
+    pub fn get_module(&self, id: ModuleIndex) -> Option<WrappedModule> {
         self.modules.get(id.as_usize()).cloned()
     }
 
-    /// Fetch a [Export] by [GlobalProcedureIndex]
+    /// Fetch a [WrapperProcedure] by [GlobalProcedureIndex], or `None` if index is invalid.
     #[allow(unused)]
-    pub fn get_procedure(&self, id: GlobalProcedureIndex) -> Option<&Export> {
-        self.modules.get(id.module.as_usize()).and_then(|m| m.get(id.index))
+    pub fn get_procedure(&self, id: GlobalProcedureIndex) -> Option<ProcedureWrapper> {
+        match &self.modules[id.module.as_usize()] {
+            WrappedModule::Ast(m) => m.get(id.index).map(ProcedureWrapper::Ast),
+            WrappedModule::Info(m) => {
+                m.get_proc_info_by_index(id.index).map(ProcedureWrapper::Info)
+            }
+        }
+    }
```

Comment:
Are these methods really unused? If so, maybe we should get rid of them for now?

Reply:
Removed. There were also a bunch of other `allow(unused)` methods, and I'm not sure why. Should we just remove them all?

Reply:
Yeah, I'm thinking it is worth removing them rather than keep updating functions that we don't currently need.

Reply:
Removed